egs/wsj/s5/steps/libs/nnet3/xconfig/basic_layers.py

```diff
@@ -1075,6 +1075,51 @@ def get_full_config(self):
         return ans
 
 
+class XconfigAuxiliaryLayer(XconfigLayerBase):
+    """This class is for lines like
+    'auxiliary name=aux dim=40'
```

Comment:
no, it is not presented in config line. I mentioned that in get_full_config function in this class.